build.bat

```diff
@@ -1,6 +1,2 @@
 @echo off
-
-.paket\paket.bootstrapper.exe 
-IF EXIST paket.lock (.paket\paket.exe restore)
-IF NOT EXIST paket.lock (.paket\paket.exe install)
-"packages\build\FAKE\tools\Fake.exe" "build\\scripts\\Targets.fsx" "cmdline=%*"
+dotnet run --project build/scripts -- %*
```

Comment:
`-c Release` mode?

Reply:
I guess you could do that, but I've never seen anyone do it. After all, the `build/scripts` project will never be "released". It's effectively a script.